server/views/pages/appointments/prePostAppointmentConfirmation.njk

```diff
@@ -145,6 +145,14 @@
                             value: {
                                 text: hearingType
                             }
+                        } if bookAVideoLinkEnabled,
+                        {
+                            key: {
+                                text: "Court video link"
```

Comment:
Just wondering if the text here and in the `prePostAppointments.njk` screen should be aligned?
"Court video link" vs "Court hearing link"